test/index-rollup-stable.spec.js

```diff
@@ -105,7 +117,21 @@ describe('rollup-plugin-license [rollup stable]', () => {
       },
     });
 
-    instance.load('lodash');
+    const moduleId = path.join(__dirname, 'fixtures', 'fake-package', 'src', 'index.js');
+    const modules = {
+      [moduleId]: {
+        renderedExports: [],
+        removedExports: [],
+        renderedLength: 10,
```

Comment:
Is there a test for `renderedLength===0`?

Reply:
I missed to add it, thanks!